sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -353,10 +356,30 @@ object DottyPlugin extends AutoPlugin {
           Def.valueStrict { scalaInstance.taskValue }
       }.value,
 
-      // We need more stuff on the classpath to run the `doc` task.
+      // Configuration for the doctool
+      resolvers ++= Seq(
+        Resolver.jcenterRepo,
+        Resolver.bintrayRepo("kotlin", "kotlin-dev"),
```

Comment:
I think we should add those only if `useScala3doc := true`

Reply:
I'll ask someone that understands how SBT works if that will work.

Reply:
ok. I will try to make it work

Reply:
What I was worried here about is when exactly the value of `useScala3doc` is checked. It has to be checked after plugin users could have already set it, otherwise it will always be false. Also whatever we do here should work correctly with users adding their own resolvers.

Reply:
The following will do what you want:
```scala
resolvers ++= {
  if (useScala3doc.value) {
    Seq(...)
  } else Nil
}
```

Reply:
@sjrd Excellent, thanks for checking up on this!

Reply:
Will include that in #10092